i-poc/ccfaas-app/load_program.sh

```diff
@@ -51,24 +52,15 @@ then
 	exit 1
 fi
 
-# Check server availability
```

Comment:
Can you remind me why we removed this check?

Reply:
Fixed on the next two commits. It seems that my rebase deleted some new updates